### ISSUE TITLE
2.5.0 release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#1982]
- #1977

**Comment by joeparsons** [comment]
## Release notes snippet

```
### New changes (not included in 2.4.x)

#### New features
- Fixes #515 Version indicator changes (#1796)
- Closes #397 - Add linkit module (#1836)
- Closes #647: Use linkit for Card field link elements. (#1880)
- Closes #574 Adding az_menu block_content type. (#1887)
- Update Arizona Bootstrap to 2.0.17 (#1928)

#### Bug fixes
- Closes #1848: Fix PHPstan errors since updating to phpstan-drupal 1.1.25. (#1849)
- Closes #1881: Fix broken Dependency check job in security.yml GH actions workflow. (#1882)
- Closes #1837: Update URL of Global Engagement link in global footer. (#1860)
- Closes #1858: Fix broken ProboCI 'Check for PHP errors and notices' step. (#1859)

#### Breaking or visual changes to existing behavior
- Closes #1864: Add new breadcrumb-item-home class to front page segment. (#1899)

#### Drupal contrib project updates
- Update drupal/xmlsitemap requirement from 1.2.0 to 1.3.0 (#1775)
- Update drupal/metatag requirement from 1.19.0 to 1.21.0 (#1765)
- Update drupal/media_library_theme_reset requirement from 1.1.0 to 1.3.0 (#1806)
- Update drupal/exclude_node_title requirement from 1.3.0 to 1.4.0 (#1808)
- Update drupal/block_class requirement from 1.3 to 2.0.0 (#1712)
- Update drupal/media_library_form_element requirement from 2.0.3 to 2.0.4 (#1809)
- Update drupal/cas requirement from 2.0.0 to 2.0.1 (#1811)
- Update drupal/viewsreference requirement from 2.0-beta2 to 2.0.0-beta3 (#1783)
- Update drupal/environment_indicator requirement from 4.0.6 to 4.0.7 (#1807)
- Update drupal/intelligencebank requirement from 3.0-beta1 to 3.0.0 (#1813)
- Update drupal/views_bootstrap requirement from 4.3 to 5.4.0 (#1815)
- Update drupal/pathauto requirement from 1.10.0 to 1.11.0 (#1812)
- Update drupal/entity_browser requirement from 2.7.0 to 2.8.0 (#1810)
- Update drupal/token requirement from 1.10.0 to 1.11.0 (#1819)
- Update drupal/jquery_ui_datepicker requirement from 1.2.0 to 1.3.0 (#1829)
- Update drupal/cas requirement from 2.0.1 to 2.0.2 (#1828)
- Closes #1816: Update drupal/externalauth requirement from 1.4.0 to 2.0.2 (#1818)
- 2022-08-31 Consolidated dependabot updates. (#1878)
- Update drupal/environment_indicator requirement from 4.0.7 to 4.0.9 (#1893)

#### Other
- Closes #1803: Allow more open dependabot PRs. (#1804)

### Backported changes (also included in 2.4.x)

#### Bug fixes
- Closes #1382: Add role_delegation patch for restricting access to cancel/remove admin users (#1773)
- Closes #1790, Closes #1792: Update Pathauto patterns for Drupal 9.3+ and use drupal/ctools:4.0.1. (#1791)
- Closes #1523 use az_entity_embed_process before passing to text_format_recognizer (#1798)
- Closes #1839 Add entity_embed_process plugin to field_az_text_area in az_paragraph_html migration. (#1840)
- Closes #1844 - Fix twig variable index for alt text on gallery images (#1846)
- Closes #1841: "Downgrade" ctools to 8.x-3.11. (#1842)
- Closes #1856: Fix deprecation warning caused by empty card/accordion body fields on PHP 8.1+. (#1857)
- Closes #1534 Adding multi-byte character support to EntityEmbedProcess migrate plugin. (#1843) 
- Fixes #1725 Disable Card Preview Links. (#1795)
- Closes #1674 Fix up links in migrations (#1800)
- Closes #1891: Use privileged GH token for 'Create Release' workflow. (#1892)
- Closes #937: Fix missing icons in accordion previews. (#1895)
- Closes #1888: Apply upstream config schema changes to easy_breadcrumb.settings.yml. (#1889)
- Closes #1903: Tag correct revision during 'Create release' GH actions workflow. (#1905)

#### Drupal core - Patch level releases
- Update drupal/core-recommended requirement from 9.4.3 to 9.4.5 (#1805)

#### Other
- Closes #1782: Add mesa as background color option to cards, text on background, and split screen. (#1784)
- Closes #177 - Adds masquerade module (#1833)
- Closes #692: Add version to info.yml during release action. (#1865)
```

**Comment by trackleft** [comment]
[2.5.0 released](https://github.com/az-digital/az_quickstart/releases/tag/2.5.0)
